lib/drawSquareDoubles.c

```diff
@@ -14,6 +13,95 @@
 #define DEBUG2 '2'
 #define DEBUG3 '3'
 
+triangle* meshInit (int numberOfTris){
+    return (triangle*) malloc(numberOfTris * sizeof(triangle));
+}
+
+void plotTrianglePoints(double arr[][3], triangle* triangles, int numberOfTris){
+
+    int totalNumberOfPoints = numberOfTris*3;
+    int count = 0;
+
+    for (int i = 0; i < numberOfTris; i++){
+        for(int j = 0; j < 3; j++){
+                
+            arr[count][0] = triangles[i].p[j].x;
+            arr[count][1] = triangles[i].p[j].y;
+            arr[count][2] = triangles[i].p[j].z;
+            count++;    
+        }
+    }
+}
```

Comment:
we're essentially just making a copy of all the points into this points array ? Why?

Reply:
In that format, made the data compatible with previously written functions. If I decide to implement translation, the maths required will be different, 4x4 matrices for example, and then this won't make much sense to do. Was really just aiming to get something to render.